phpBB/includes/ucp/ucp_register.php

```diff
@@ -257,6 +270,19 @@ function main($id, $mode)
 					$error[] = $user->lang['NEW_PASSWORD_ERROR'];
 				}
 			}
+			/**
+			* Check UCP registration data after they are submitted
+			*
+			* @event core.ucp_register_data_after
+			* @var	bool	submit		Do we display the form only
+			*							or did the user press submit
+			* @var	array 	data		Array with current ucp registration data
+			* @var	array	cp_data		Array with custom profile fields data
+			* @var	array 	error		Array with list of errors
+			* @since 3.1.4-RC1
+			*/
+			$vars = array('submit', 'data', 'cp_data', 'error');
```

Comment:
OK, @marc1706.
Could you explain why phpBB uses `@change` and `@changed` invalid tags instead of several `@since` tags please?
Maybe there is a guideline on this case? Because there is still difference in tags usage when somebody modifies an event.
Official documentation http://www.phpdoc.org/docs/latest/references/phpdoc/tags/since.html suggests to add more than one `@since` tag.
> This tag can occur multiple times within a PHPDoc. In that case, each occurrence is treated as an entry to a change log. It is RECOMMENDED that you also provide a description to each such tag.